docs/uml-diagrams/FindCommand.puml

```diff
@@ -0,0 +1,80 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+skinparam classAttributeIconSize 0
+
+abstract class Command
+class Inventory
+class FindCommand
+class Parser
+class Ui
+
+Command <|. FindCommand
+Inventory <.. FindCommand
+Ui <.. FindCommand
+FindCommand <.. Parser
+
+interface Command {
+    +execute():void
+    +isExit():boolean
+}
+
+class FindCommand {
+    -name:String
+    +FindCommand(name:String)
+    +execute():void
+    +isExit():boolean
+}
+
+class Inventory {
+    {static}+getItems():items: ArrayList
```

Comment:
It is to my understanding that we do not need to give a name to the return type of the method. I suggest using this instead.
```suggestion
    {static}+getItems():ArrayList<Item>
```